paf-mvp-core-js/src/endpoints.ts

```diff
@@ -12,6 +12,7 @@ const jsonRead = '/v1/ids-prefs';
 const jsonWrite = '/v1/ids-prefs';
 const jsonVerify3PC = '/v1/3pc';
 const jsonNewId = '/v1/new-id';
+const jsonSeed = '/v1/seed';
```

Comment:
[NIT] to be more consistent with other endpoint names, I would use an action name, such as "buildSeed"

Reply:
Done.